backend/blueprint/api/instance/config.py

```diff
@@ -0,0 +1,66 @@
+import os
+from dotenv import load_dotenv, find_dotenv
+
+load_dotenv(find_dotenv())
+
+
+DEBUG = os.getenv('DEBUG')
+
+POSTGRES = {
+    'user': os.getenv('POSTGRES_USER'),
+    'pw': os.getenv('POSTGRES_PASSWORD'),
+    'host': os.getenv('POSTGRES_HOST'),
+    'port': os.getenv('POSTGRES_PORT'),
+    'db': os.getenv('POSTGRES_DATABASE'),
+    'secret': os.getenv('POSTGRES_SECRET')
+}
+
+
+class MailConfig(object):
```

Comment:
Please explain the use of it :+1:

Reply:
it's here because of https://github.com/fossasia/badgeyay/pull/712